backend/apid/graphql/mutations_test.go

```diff
@@ -5,7 +5,6 @@ import (
 	"testing"
 
 	"github.com/sensu/sensu-go/backend/apid/graphql/globalid"
-	client "github.com/sensu/sensu-go/backend/apid/graphql/mockclient"
 	"github.com/sensu/sensu-go/backend/apid/graphql/schema"
 	"github.com/sensu/sensu-go/types"
```

Comment:
Looks like there's a lot more aliasing we can change to get to our consistent, desired pattern